meshapp/src/pages/Profiles/ProfilePage.tsx

```diff
@@ -294,6 +294,71 @@ const ProfileEducation = (props: { education: Education }) => {
     `${inputName} cannot have whitespace at beginning or end.`;
   const charLimit = (inputName: string) => (val: string) =>
     val.length < 100 || `${inputName} cannot be longer than 100 characters.`;
+
+  //state for education array
+  //for now id will just be index
+  const [educationData, setEducationData] = useState(
+    props.education.map((currentEd, index) => {
+      return {
+        accordionId: index,
+        comboOneVal: currentEd.degree,
+        comboTwoVal: currentEd.school,
+        descText: currentEd.description,
+      };
+    })
+  );
+
+  //Edit and Delete education accordion handlers,
+  // takes in id number and returns a function that edits/deletes an education accordion with that id
+
+  //The accordions already handle editing in terms of local changes without saving,
+  //so this edit handler is only for saving on the backend
+
+  //currently does nothing but will be tied to the edit idit icon
```

Comment:
I dont think there should be comments that say it does nothing. The "insert x thing needed here" kind of comments should be enough. Comments like these are clutter and may never be deleted appropriately in the future

Reply:
got it, I'll remove them now